src/appleseed/renderer/modeling/aov/cryptomatteaov.cpp

```diff
@@ -510,11 +510,17 @@ namespace
                         pixel_values.push_back(b);
 
                         // Remove background contribution.
+                        size_t ranked_vector_start = 0;
+                        size_t ranked_vector_size = ranked_vector.size();
+
```

Comment:
I'd suggest to remove this blank line so that the comment above applies to the whole block.